app/views/pages/volunteer.html.haml

```diff
@@ -1,5 +1,5 @@
 .row
-  = image_tag 'home_banner_1_placeholder.png'
+  = image_tag "","data-interchange" => "[home_banner_1_placeholder.png, large],[home_banner_1_placeholder.png, only screen and (min-width: 40.063em) and (max-width: 64em)],[home_banner_1_placeholder.png, only screen and (max-width: 40em)]"
```

Comment:
We should be using `image_path` (a Rails helper) to generate proper image paths. Here's a good example with interchange: https://coderwall.com/p/uip7xg/ruby-on-rails-image-helper-for-retina-display-with-foundation-interchange.
In that example they used a helper to create a retina image, but we could have a helper that automatically generates `_small`, `_medium` and `large` image paths, and then we'd have to ensure that we always have images for all 3 sizes.